# Apriltag
tag_id: 0
apriltag_frame: "tag_0"

# Dock position offset from pallet position
x_offset: 0.5 # m
y_offset: 0.0 # m
yaw_offset: 0 # deg

# Dock position tolerance
goal_xy_tolerance: 0.025 # m
goal_yaw_tolerance: 5 # deg

# Distance tolerance to filter out outlier dock position
inlier_dist: 0.2 # m
inlier_yaw: 10 # deg

# Number of frames tracked to determine the average dock position
frames_tracked: 5
